### ISSUE TITLE
license statement in package metadata misleading/incomplete

### ISSUE THREAD

**Issue by jamin-aws-ospo** [ckeditor/ckeditor4#4567]
## Type of report

Task

## Provide description of the task

The [licensing information listed in the package metadata](https://github.com/ckeditor/ckeditor4/blob/major/package.json#L43) states:
```
"license": "(GPL-2.0-or-later OR LGPL-2.1 OR MPL-1.1)"
```

However, the [LICENSE.md](https://github.com/ckeditor/ckeditor4/blob/major/LICENSE.md) indicates:
```
GNU General Public License Version 2 or later (the "GPL") https://www.gnu.org/licenses/gpl.html (See Appendix A)

GNU Lesser General Public License Version 2.1 or later (the "LGPL") https://www.gnu.org/licenses/lgpl.html (See Appendix B)

Mozilla Public License Version 1.1 or later (the "MPL") https://www.mozilla.org/MPL/MPL-1.1.html (See Appendix C)
```

Note the **or later** on each of the licenses.

Additionally, there are components under different licenses:
* [samples/toolbarconfigurator/font/LICENSE.txt](https://github.com/ckeditor/ckeditor4/blob/major/samples/toolbarconfigurator/font/LICENSE.txt)
* [samples/toolbarconfigurator/lib/codemirror/LICENSE](https://github.com/ckeditor/ckeditor4/blob/major/samples/toolbarconfigurator/lib/codemirror/LICENSE)
* [plugins/codesnippet/lib/highlight/LICENSE](https://github.com/ckeditor/ckeditor4/blob/major/plugins/codesnippet/lib/highlight/LICENSE)

The above should not be taken as an exhaustive list, it's just what I found by spot checking.

*What steps should be taken to fulfil the task?*
Correct/clarify the licensing terms of the project/package.

## Other details
N/A

**Comment by sculpt0r** [comment]
Hi @jamin-aws-ospo 
Thank you for this report.

**Comment by jacekbogdanski** [comment]
@jamin-aws-ospo thank you for letting us know about this inconsistency. I've updated `package.json` license description in https://github.com/ckeditor/ckeditor4/commit/adf2f91e288ff232b1bef18e80b6ba75df2988aa These changes will be available since 4.17.0 release.

In the case of mentioned other components with different licenses, I see that all of them are covered in https://github.com/ckeditor/ckeditor4/blob/major/LICENSE.md

It's rather impossible to include every single license into `package.json` file alone and keep it readable. LICENSE.md file should be always checked to confirm license compatibility with your application. Mentioned components are a minority of the whole CKEditor 4 codebase and `package.json` already includes licenses for most of our code.
